packages/lodestar/src/network/reqResp.ts

```diff
@@ -245,52 +241,37 @@ export class ReqResp extends (EventEmitter as IReqRespEventEmitterClass) impleme
     }
   }
   private async sendRequest<T extends ResponseBody>(
-    peerInfo: PeerInfo,
+    peerId: PeerId,
```

Comment:
Can we keep this as `PeerInfo`?
It looks like `dialProtocol`, below should work with a PeerInfo too. https://github.com/libp2p/js-libp2p/blob/master/src/index.js#L282

Reply:
yeah I changed it back to PeerInfo across interfaces.